scripts/analyze_telemetry.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e;
```

Comment:
Can we do this with python?

Reply:
Yeah, but it would require more effort

Reply:
Yes, I understand, but in the long run it will be a lot easier to maintain.

Reply:
It is just 50 lines of code for development. If it will need to grow, then we could rewrite it. It is not some obscure code, it is quite readable and understandable